docs/static/spec/openapi/logstash-api.yaml

```
@@ -27,19 +27,34 @@ info:
     label: Feedback
     url: https://github.com/elastic/docs-content/issues/new?assignees=&labels=feedback%2Ccommunity&projects=&template=api-feedback.yaml&title=%5BFeedback%5D%3A+
 servers:
-  - url: /
+  - url: "{scheme}://{address}:{port}"
+    variables:
+      scheme:
+        default: http
+        enum:
+          - http
+          - https
+      address:
+        default: localhost
+        description: the hostname or ip address of your logstash instance
+      port:
+        default: 9600
```

```suggestion
        default: "9600"
```

swagger said default must be a string